source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,306 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page, you can learn how to query **geospatial data**
+with the MongoDB Java Driver. 
+
+Geospatial data is data tied to a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
```

Comment:
Thanks for the note. After rereading this a few times, I can see how it doesn't add much for readers. I rewrote the introductory sentence to try and communicate the same message as this list. Fix:

Responding to feedback around "should... if you would like to": This is the phrasing Chris and I arrived at after a few meetings around the [Sorts Page](https://docs-mongodbcom-staging.corp.mongodb.com/a54a90a/java/docsworker-xlarge/DOCSP-9562-Geo/fundamentals/builders/sort/) (first page I wrote that received high volume of notes and had a few synchronous meetings). If I remember correctly, this passive phrasing was considered desirable as we are discussing the reader's behavior (it could be rude to be assertive in saying "you must do"). I personally agreed with this stance of being passive when discussing the reader's behavior, but I am totally fine not using this phrasing.

I think this would be a good point of discussion in a team PR review meeting.

Reply:
In my final proofread, I ended up changing this to https://github.com/mongodb/docs-java/pull/78/commits/e897a9d9371c9c437a67590e7e4eab524684e834. I think changing the top heading to "search **geospatial data** leaves work for the list at the bottom of the intro to break down the concrete concepts we will cover on the page later on.

Let me know if you disagree, and as this is a comment that it not intuitive to me, it may be worth discussing it synchronously.